runtime/bindings/python/initialize_module.cc

```diff
@@ -16,6 +16,14 @@
 #include "iree/base/internal/flags.h"
 #include "iree/hal/drivers/init.h"
 
+namespace {
+// Stable storage for flag processing.  Flag handling uses string views,
+// expecting the caller to keep the original strings around for as long
+// as the flags are in use.  This object holds one set of flag strings
+// for each invocation of parse_flags.
+std::vector<std::vector<std::string>> alloced_flag_cache;
```

Comment:
Needs to contain an allocated vector either by unique_ptr or raw pointer (the former is friendlier to unloading). How you have it, the inner vector will not have pointer stability when the outer vector is resized, so all of the contained strings will also not be pointer stable. Just change the inner vector to a unique_ptr and heap allocate it.

Reply:
Oh, yes, this is true.  Will fix

Reply:
Probably not getting enough sleep, so very glad to have you watching my back!  Now that I've thought about it some more, `std::vector<std::vector<std::string>>` was probably safe enough.  When the outer vector's data array gets expanded, the inner vectors should be getting move constructed, preserving their own data arrays.  Anyway, reimplemented using `unique_ptr`.